source/_integrations/bayesian.markdown

```diff
@@ -185,3 +185,67 @@ binary_sensor:
 ```
 
 {% endraw %}
+
+## Changes in 2022.10
```

Comment:
We don't keep t a changelog in our documentation. For this we have our release notes and the breaking changes section.

Reply:
It was suggested by @dgomes here https://github.com/home-assistant/core/issues/79694#issuecomment-1272404970

I am happy to leave in issues and the forum. I'll convert to draft and leave you two to discuss (if there is any discussion to be had).

That being said - the new repairs UI has `learn_more_url` argument/button. So there is probably a need for somewhere to store repairs/deprecation/breaking/migration information.

Reply:
> It was suggested by @dgomes here https://github.com/home-assistant/core/issues/79694#issuecomment-1272404970

> I am happy to leave in issues and the forum. I'll convert to draft and leave you two to discuss (if there is any discussion to be had).

🤷 That is cool, but again, we don't do that. So, not much to discuss.

Again: If the provided breaking changes section in the release notes (which you provided) are not extensive enough, please feel free to extend those.

Otherwise, we can close this PR.

Reply:
> Again: If the provided breaking changes section in the release notes (which you provided) are not extensive enough, please feel free to extend those.

How do I edit those. I did ask on the discord during pre-release but no-one knew where they are stored.

Reply:
I was pointing out the usefulness of an example including explanation into the documentation.

I believe that falls into the scope of the documentation.